### DISCUSSION TITLE
How can i work out percentage from fields?

### DISCUSSION THREAD

**Discussion by MonsterITServices** [YetiForceCompany/YetiForceCRM#16763] [How to] [answered]
HI, I've added a three fields to projects. 
1. customer charged
2. cost to us
3. total profit

How can i get a field which works out profit margin (in percentage). 
So field 1 is what we charge then subtract field 2 and output in field 3, then in another field give the percentage?

Does anyone know how to do this?

Thank you in advance.

**Comment by s2b-git** [comment, marked as answer]
Workflows. Make sure fields are currency type and play with basic formulas such as: (sum_gross/1.07)*0.03

**Reply by MonsterITServices** [reply]
Perfect thank you.
